mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -323,14 +323,16 @@ public void testInsertSupportedTypes() throws IOException {
         continue;
       }
       String columnName = type.typeId().toString().toLowerCase() + "_column";
+      String tableName = type.typeId().toString().toLowerCase() + "_table_" + i;
 
       Schema schema = new Schema(required(1, "id", Types.LongType.get()), required(2, columnName, type));
       List<Record> expected = TestHelper.generateRandomRecords(schema, 5, 0L);
 
-      Table table = testTables.createTable(shell, type.typeId().toString().toLowerCase() + "_table_" + i,
-          schema, PartitionSpec.unpartitioned(), fileFormat, expected);
+      Table table = testTables.createTable(shell, tableName, schema, PartitionSpec.unpartitioned(), fileFormat,
+          expected);
 
       HiveIcebergTestUtils.validateData(table, expected, 0);
```

Comment:
Now that we have a way of validating data via SQL as well, can we somehow reflect the difference that `validateData` queries that data via Iceberg API? Maybe through the method names or their respective javadocs

Reply:
updated javadoc, and renamed the method